ui/components/app/confirm-page-container/confirm-page-container.component.js

```diff
@@ -229,9 +240,70 @@ export default class ConfirmPageContainer extends Component {
               ethGasPriceWarning={ethGasPriceWarning}
               hideTitle={hideTitle}
               supportsEIP1559V2={supportsEIP1559V2}
+              currentTransaction={currentTransaction}
+              nativeCurrency={nativeCurrency}
+              networkName={networkName}
+              showBuyModal={showBuyModal}
             />
           )}
-          {shouldDisplayWarning && (
+          {shouldDisplayWarning && errorKey === INSUFFICIENT_FUNDS_ERROR_KEY && (
+            <div className="confirm-approve-content__warning">
+              {currentTransaction.chainId === '0x1' ? (
+                <ActionableMessage
+                  message={
+                    <Typography
+                      variant={TYPOGRAPHY.H7}
+                      align="left"
+                      margin={[0, 0]}
+                    >
+                      {this.context.t('insufficientCurrency', [
+                        nativeCurrency,
+                        networkName,
+                      ])}
+                      <button
```

Comment:
Any particular reason why you favor html button to the Button component in ui package?

Reply:
Yeah, I think using the `<Button>` component is a good idea

Reply:
updated.